scripts/e2e-test.sh

```diff
@@ -103,15 +105,16 @@ while [ "$#" -gt 0 ]; do
     --onfail)
         shift
         case $1 in
-            continue)
+            uninstall)
                 on_fail=$1
                 ;;
             stop)
                 on_fail=$1
                 ;;
             *)
+                echo "invalid option for --onfail"
                 help
-                exit 2
+                exit 1
```

Comment:
It would be nice to have symbols for these exit values

Reply:
Done